core-java-modules/core-java-arrays-multidimensional/src/test/java/com/baeldung/array/minmax/MinMax2DArrayUnitTest.java

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.array.minmax;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.IntSummaryStatistics;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MinMax2DArrayUnitTest {
+
+    @Test
+    public void givenArray2DWhenFindMinAndMaxUsingForLoopsThenCorrect() {
+        int[][] array = {{8, 4, 1}, {2, 5, 7}, {3, 6, 9}};
+
+        int min = array[0][0];
+        int max = array[0][0];
+        for (int[] row : array) {
+            for (int currentValue : row) {
+                if (currentValue < min) {
+                    min = currentValue;
+                }
+                else if (currentValue > max) {
```

Comment:
Quick format adjustment.
```suggestion
                } else if (currentValue > max) {
```